### DISCUSSION TITLE
subfolder defaults

### DISCUSSION THREAD

**Discussion by aegisnir** [BlafKing/sd-civitai-browser-plus#96] [Ideas]
Is there any way to use a subfolder for all downloads by default?  Currently, if I find a model I want to download, I have to manually select the subfolder option.  I would love to be able to just click download without having to select the subfolder options as I will never not want my models in sub folders.  Perhaps this is more a feature request?  I was hoping there was something I could do in the extension files to perhaps set that to the default if its not included in the UI?

**Comment by BlafKing** [comment]
Nice suggestion!
This should be possible but may be a bit tricky to implement since, each different model type `checkpoint, Lora, textual inversion etc..` has different available sub folders, I'll have a look to see if I can properly implement this! :)

**Comment by BlafKing** [comment]
This has been implemented since v1.15.2! :)

**Comment by aegisnir** [comment]
Wow thank you!  I cant believe you did that in just a couple weeks :).  Is there any way to get the extension to simply use the existing naming convention you have and just set that to the default though?  For example, if you want to download "Ink scenery | 水墨山水", you can choose "/opt/stable-diffusion/data/models/Lora/Ink scenery | 水墨山水/v1.0" as the subfolder to download the model to manually.  Can we set this naming convention as the default to make a folder as "/model/version" when downloading so we don't have to manually select it from the dropdown each time?
<img width="1686" alt="Screenshot 2023-11-22 094319" src="https://github.com/BlafKing/sd-civitai-browser-plus/assets/43301130/61c6c759-7c05-48bf-8cb0-65b90aa0734c">
<img width="207" alt="Screenshot 2023-11-22 094359" src="https://github.com/BlafKing/sd-civitai-browser-plus/assets/43301130/5cc182d4-a774-4f67-b68d-a1fecac130b3">
